app.py

```diff
@@ -100,6 +104,14 @@ def process_import_function(event, context):
 
         try:
             do_import(juris["id"], f"{datadir}{abbreviation}")
+            stats.send_last_run(
+                "last_collection_run_time",
+                {
+                    "jurisdiction": juris["name"],
+                    "scrape_type": "import",
+                },
+            )
+
         except Exception as e:
             logger.error(
                 f"Error importing jurisdiction {juris['id']}: {e}"
```

Comment:
We want to force a batch send (`stats.close()`) before the lambda terminates or we'll miss some data. Also, we might consider a `objects_collected{jurisdiction=<>,session=<>}` counter to indicate how many objects are being collected this way. There is a similar metric in the regular ingest that you could use as reference.

Reply:
Thanks @johnseekins - this stats is still being collected by the scraper, that's why I didn't add it to the lambda, the stat added here is excepted in os-core when realtime is activated. If we do this we'd be duplicating the stat.

Reply:
Okay. If stats are still being produced, that's great. We'll still want to send `stats.close()` to ensure all batched stats are sent at the end of the lambda function.

Reply:
Yeah, will add that now.